src/_component/skills/skills.pug

```diff
@@ -0,0 +1,98 @@
+-
+  var headerName = 'Skills';
+  var tableName = 'Legend';
+  var frontEndList = [
+    {
+      title: 'Front End',
+      skillList: [
+        'HTML5',
+        'CSS3',
+        'JavaScript',
+        'React',
+        'jQuery',
+        'JSON',
+        'SASS',
+        'Bootstrap'
+      ]
+    }
+  ]
+  var backEndList = [
+    {
+      title: 'Back End',
+      skillList: [
+        'PUG',
+        'GIT',
+        'NodeJS',
+        'MySQL',
+        'Less'
+
+      ]
+    }
+  ]
+  var otherList = [
+    {
+      title: 'Other',
+      skillList: [
+        'JADE',
+        'Gulp',
+        'JSON',
+        'Windows'
+      ]
+    }
+  ]
+
+#skills
+.skills-box
+    h1.skills-title=headerName
+.skills-section
+    .space
+    button.youtube-button
+        a.link-access(href="https://www.youtube.com")
+            span.youtube-icon(class="my-cv-youtube")
+    .space
+
+    h2.legend-title=tableName
+    .skill-table
```

Comment:
Why you got `skills` in your class when should be only `skill`? When you create classes, use the singular word, it will be easier for you in future.